docs/docs/sign-in-out.md

```diff
@@ -23,17 +36,29 @@ server.auth.sign_out()
     limited by the maximum session length (of four hours) on Tableau Server.
 </div>
 
-
-Alternatively, for short programs, consider using a `with` block:
+### Disabling certificate verification and warnings
+If a self-signed certificate is used, certificate verification can be disabled with ```add_http_options```:
 
 ```py
 import tableauserverclient as TSC
 
 tableau_auth = TSC.TableauAuth('USERNAME', 'PASSWORD')
 server = TSC.Server('http://SERVER_URL')
+server.add_http_options({'verify': False})
+```
 
-with server.auth.sign_in(tableau_auth):
-    # Do awesome things here!
+However, each subsequent REST API call will print an ```InsecureRequestWarning```:
+```
+C:\Program Files (x86)\Python35-32\lib\site-packages\requests\packages\urllib3\connectionpool.py:838: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/security.html
+InsecureRequestWarning)
 ```
 
-The TSC library signs you out of Tableau Server when you exit out of the `with` block.
+These warnings can be disabled with the following line:
+```py
+requests.packages.urllib3.disable_warnings(InsecureRequestWa‌​rning)
+```
+
+### A better way to avoid certificate warnings
+Instead of disabling warnings or certificate verification when using self-signed certificates, we recommend using a Certificate Authority (CA) signed certificate.
```

Comment:
See my note about the causes -- it could be signed by an internal CA but your client is misconfigured, etc (I think) but otherwise this section looks pretty solid.